src/api/.env

```diff
@@ -0,0 +1,4 @@
+SCICRUNCH_API_URL=https://scicrunch.org/api/1/term/elastic/search
+SCICRUNCH_API_KEY=tvVqyYwrQqolqVfMo45cu31t5uEGx6RZ
```

Comment:
@jrmartin the key should be injected during the build through codefresh, we cannot have this in the codebase. Thanks

Reply:
We can achieve the injection if we use an .env variable as well, would work for both locally and in codefresh. If you prefer though I can remove it and have these as part of codefresh for deployment, and environment variables when running locally @ddelpiano